client/components/catalog/Card/index.vue

```diff
@@ -46,16 +46,35 @@
           mdi-pin
         </v-icon>
       </v-btn>
+      <v-col cols="12" sm="8" md="9" class="pa-0">
```

Comment:
You should just flex the container, keep pin space with fixed with and make this one flex:1 (rather then using col 😉 )